mypage/models.py

```diff
@@ -0,0 +1,66 @@
+from django.db import models
+
+from account.models import User
+from product.models import Product
+from order.models   import Order
+
+class Mileage(models.Model):
+    point = models.IntegerField()
```

Comment:
```suggestion
    point = models.IntegerField(default=0)
```
포인트는 혹시 음수로도 변하나요? 아니라면 PositiveIntegerField라는 것도 있으니 참고하세요.
그리고 인티저 필드등을 쓸때는 보통 디폴트 값을 설정해주세요.